signature-pad/src/main/java/com/github/gcacace/signaturepad/utils/SvgPathBuilder.java

```diff
@@ -14,14 +14,13 @@
     private final Integer mStrokeWidth;
     private final SvgPoint mStartPoint;
     private SvgPoint mLastPoint;
-    private Character mLastSvgCommand;
 
     public SvgPathBuilder(final SvgPoint startPoint, final Integer strokeWidth) {
         mStrokeWidth = strokeWidth;
         mStartPoint = startPoint;
         mLastPoint = startPoint;
-        mLastSvgCommand = null;
         mStringBuilder = new StringBuilder();
+        mStringBuilder.append(SVG_RELATIVE_CUBIC_BEZIER_CURVE);
```

Comment:
Added the relative curve command at the start of the path rather than making this decision as each point is added